themes/default/content/community/developer-advocates/matt-stratton.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Matt Stratton

```

Comment:
```suggestion
title: Matty Stratton

```
Yes, I know, I'm the one who made the original file with the name this way...lol